vm/CMakeLists.txt

```diff
@@ -27,8 +35,16 @@ endif()
 
 if(${BPFTIME_LLVM_JIT})
   set(BPFTIME_BPF_RUNTIME_NAME "bpftime_llvm_vm")
-else()
+  message(STATUS "The runtime is ${BPFTIME_BPF_RUNTIME_NAME}")
```

Comment:
Does `BPFTIME_BPF_RUNTIME_NAME` still make sense after enabling both llvm and ubpf? Should this message be updated?